src/ui-lib/sass/05-components/03-organisms/00-page-structure/_00-page-header.scss

```diff
@@ -56,12 +56,7 @@ body > header {
     display: block;
     max-height: 33px; // Otherwise IE11 makes it 150px tall :-(
     fill: $grav-co-bg;
-
-    // Bump up logo size
-    @media (min-width: gravy-breakpoint(medium)) {
-      height: 2.5rem;
```

Comment:
This change belongs to your other PR. Please revert this one, otherwise things will get messy or overwritten when this gets merged.